<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qperfeditframe</class>
 <widget class="QFrame" name="qperfeditframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="btnUndo">
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/undo.xpm</normaloff>:/pixmaps/pixmaps/undo.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRedo">
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/redo.xpm</normaloff>:/pixmaps/pixmaps/redo.xpm</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnZoomOut">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/zoom.xpm</normaloff>:/pixmaps/pixmaps/zoom.xpm</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnZoomIn">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/zoom.xpm</normaloff>:/pixmaps/pixmaps/zoom.xpm</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Snap</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbGridSnap">
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoop">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="qseq64.qrc">
         <normaloff>:/pixmaps/pixmaps/loop.xpm</normaloff>:/pixmaps/pixmaps/loop.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>12</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="Line" name="lineExpand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCollapse">
         <property name="maximumSize">
          <size>
           <width>39</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/collapse.xpm</normaloff>:/pixmaps/pixmaps/collapse.xpm</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExpand">
         <property name="maximumSize">
          <size>
           <width>39</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/copy.xpm</normaloff>:/pixmaps/pixmaps/copy.xpm</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExpandCopy">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>39</width>
           <height>25</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="qseq64.qrc">
           <normaloff>:/pixmaps/pixmaps/expand.xpm</normaloff>:/pixmaps/pixmaps/expand.xpm</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>11</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vbox_centre"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qseq64.qrc"/>
 </resources>
 <connections/>
</ui>
